.buildkite/scripts/pipelines/security_solution_quality_gate/api_integration/api-integration-tests.sh

```diff
@@ -90,7 +90,9 @@ FORMATTED_KB_URL="${KB_URL/https:\/\//}"
 # This is used in order to wait for the environment to be ready.
 sleep 150
 
-TEST_CLOUD=1 TEST_ES_URL="https://elastic:$PASSWORD@$FORMATTED_ES_URL:443" TEST_KIBANA_URL="https://elastic:$PASSWORD@$FORMATTED_KB_URL:443" yarn run $1
+echo "uername: $USERNAME"
```

Comment:
Was this intended?

Reply:
Hey, I'm sorry, I meant to open as draft and then I got pulled into something else. I'll clean up right now.